### ISSUE TITLE
hi tried it but error on checkout conf..im using windows 7

### ISSUE THREAD

**Issue by magboo** [brandonrninefive/SeleniumShoeBot#1]

**Comment by brandontruggles** [comment]
Hello, can you please screenshot the error for me? I need more information in order to help.

**Comment by magboo** [comment]
im actaully using python 3.0 
![capture](https://cloud.githubusercontent.com/assets/25304953/22588982/6860be84-ea08-11e6-94cf-d1d75fc7efac.JPG)

**Comment by brandontruggles** [comment]
Hmm, well for starters, the script was made for Python 2.X, so I don't believe Python 3 will work. If you still get the error after using Python 2.X, let me know.

**Comment by brandontruggles** [comment]
Closing this issue until I know that it still occurs on Python 2.X.

UPDATE: Aside from you not using Python 3, I have figured out what causes this particular error. You must add the directory that you download the script into to your PYTHONPATH environment variable. I have updated the GitHub README to now include detailed installation instructions for this project. Thank you for opening the issue!
